pootle/settings/30-site.conf

```diff
@@ -15,6 +15,8 @@ MANAGERS = ADMINS
 
 DEBUG = False
 
+CANONICAL_URL = ""
```

Comment:
re Sites - using it is undocumented -and has no ui - im figuring anyone using sites is already customizing heavily - probs noone

i am wondering tho if there is any possibility of a historical use that might mean someone has `sites` configured